packages/next/src/lib/get-files-in-dir.ts

```diff
@@ -0,0 +1,22 @@
+import { join } from 'path'
+import fs from 'fs/promises'
+import type { Dirent, StatsBase } from 'fs'
+
+export async function getFilesInDir(path: string): Promise<string[]> {
```

Comment:
I renamed this to something more fitting, since I couldn't really understand what `flatReaddir` was supposed to reference unless historically it was recursive and did indeed flatten something.